group.py

```diff
@@ -2,4 +2,49 @@
 
 # Your code to go here...
 
-my_group =
+# define individual people
+
+liam = {'Name': 'Liam',
+        'Age': 22,
+        'Job': 'Student (MRes Medical Imaging)',
+        'Connections': [['Ed', 'Colleague'],
```

Comment:
Here we have connections defined as an array, where the first element is the name and the second element is the colleage. In some other solutions, connections is a list of dict's, where each dict contains a "name" field giving the name, and a "type" field that stores the type of the relationship. It might be worth considering the advantages and disadvantages of each approach - this works perfectly fine but we thought that maybe using a dictionary for each relation makes it easier to avoid accidental issues like putting the name and the relation type in the wrong order in the list.